src/components/util/storybook.js

```diff
@@ -109,3 +111,43 @@ export const storyBackgrounds = {
   check:
     'url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAcAAAAHCAYAAADEUlfTAAAAHElEQVQYV2M8fPiwLwMOwAiStLW13YxNftBJAgAx2BqeI9XcBAAAAABJRU5ErkJggg==)',
 };
+
+export const ImportPath = ({ component, section }) => {
+  const [, toggleHover] = useToggleHover();
```

Comment:
minor question - if you only need the second index of a hook return value, is it better to just do this?

```
const toggleHover = useToggleHover()[1];
```

Reply:
Yeah it's a good question. I feel like a developer could get more easily confused at that, given the overwhelming pattern of hooks to de-structure the array? I can update though... don't personally feel strongly about either way (trade off IMO)